pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -285,7 +285,8 @@ protected void updatePhaseTimingForTables(Set<String> tableNames, BrokerQueryPha
    * @return true if the query was successfully cancelled, false otherwise.
    */
   protected abstract boolean handleCancel(long queryId, int timeoutMs, Executor executor,
-      HttpClientConnectionManager connMgr, Map<String, Integer> serverResponses) throws Exception;
+      HttpClientConnectionManager connMgr, Map<String, Integer> serverResponses)
```

Comment:
White space change. Can it ber reverted ?

Reply:
This was not intentional but it aligned with the pinot formatting so it should be okay.